src/Compilers/Core/Portable/Diagnostic/DiagnosticSeverity.cs

```diff
@@ -27,7 +27,12 @@ public enum DiagnosticSeverity
         /// <summary>
         /// Something not allowed by the rules of the language or other authority.
         /// </summary>
-        Error = 3
+        Error = 3,
+
+        /// <summary>
+        /// Something that can be potentially improved.
+        /// </summary>
+        Suggestion = 4,
```

Comment:
FWIW, I think there are a couple of problems renaming "Info" to "Suggestion" in the UI.

1. It's potentially more confusing since it wouldn't map to the actual diagnostic severity.
2. Any existing info diagnostics would suddenly be highlighted in the editor. In a sense, that'd be a breaking change for existing diagnostic authors.

There has been a ton of design discussion about this already. However, looping in @kuhlenh, who's been driving the design of this feature.